src/components/Articles.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from "react";
+import { fetchArticles } from "./utils/utils";
+import { AiFillLike } from "react-icons/ai";
+
+const Articles = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    fetchArticles()
+      .then(({ articles }) => {
+        setArticles(articles);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }, []);
```

Comment:
Nice, but we should not need to de-structure articles, this should simply be sent down as the value to access directly.

Instead of just console.log'ing the error if there is one. How would you be able to show the error if it does error?